build.gradle

```diff
@@ -1,17 +1,17 @@
 buildscript {
     ext {
-        compose_version = '1.1.0-alpha06'
+        compose_version = '1.3.0-beta02'
     }
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.4"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31"
-        classpath "com.google.dagger:hilt-android-gradle-plugin:2.38.1"
-        classpath 'com.google.gms:google-services:4.3.10'
-        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.8.1'
+        classpath "com.android.tools.build:gradle:7.2.2"
```

Comment:
Incidentally, I also found out today that `@AddTrace` does not work with AGP 7.2+, i.e. firebase-perf gradle plugin's code instrumentation does not work.

Reply:
Seems like a known issue since AGP 7.1: https://developer.android.com/studio/releases/gradle-plugin#upgrade-firebase-compatibility

Reply:
It's similar, but different. There is version 1.4.1 that works with AGP 7.1, but 1.4.2 is not released yet(that will support AGP 7.2+)